android/app/src/main/java/org/fossasia/openevent/data/facebook/Comments.java

```diff
@@ -5,16 +5,10 @@
  */
 import java.util.ArrayList;
 
+import lombok.Data;
+
+@Data
 public class Comments {
 
     private ArrayList<CommentItem> data;
```

Comment:
Someone please open the issue to change this class to just List<CommentItem> wherever this is called